test/system/260-sdnotify.bats

```diff
@@ -503,4 +505,79 @@ none | false | false | 0
 
     run_podman rmi $(pause_image)
 }
+
+@test "podman pull - EXTEND_TIMEOUT_USEC" {
+    # Make sure that Podman extends the start timeout via DBUS when running
+    # inside a systemd unit (i.e., with NOTIFY_SOCKET set).
+    image_on_local_registry=localhost:${PODMAN_LOGIN_REGISTRY_PORT}/name:tag
+    registry_flags="--tls-verify=false --creds ${PODMAN_LOGIN_USER}:${PODMAN_LOGIN_PASS}"
+    start_registry
+
+    export NOTIFY_SOCKET=$PODMAN_TMPDIR/notify.sock
+    _start_socat
+
+    run_podman push $registry_flags $IMAGE $image_on_local_registry
+    run_podman pull $registry_flags $image_on_local_registry
+    run cat $_SOCAT_LOG
+    # The 'echo's help us debug failed runs
+    echo "socat log:"
+    echo "$output"
+    is "$output" "EXTEND_TIMEOUT_USEC=30000000"
```

Comment:
It has taken me this long just to understand this one line. Wow, this is tricky! LGTM, but wow.